TestProjects/VisualEffectGraph_HDRP/Assets/ReferenceImages/Linear/WindowsEditor/Direct3D12/None/35_ShaderGraphGenerationFTP.png.meta

```diff
@@ -80,11 +68,7 @@ TextureImporter:
     maxTextureSize: 2048
     resizeAlgorithm: 0
     textureFormat: -1
-<<<<<<< HEAD
-    textureCompression: 0
-=======
     textureCompression: 1
->>>>>>> 8a20d46ea1d26dee99c8292d0d16d43d83f7b5e3
     compressionQuality: 50
     crunchedCompression: 0
     allowsAlphaSplitting: 0
```

Comment:
Where did this come from?  Weird merge artifact?  (we didn't really have a meta file with git conflicts in it, did we?)

Reply:
Where is this coming from? Hmm. Weird. I'll make sure those are cleaned up.